well-architected/mission-critical/mission-critical-deployment-testing.md

```diff
@@ -37,6 +37,10 @@ This design area focuses on how to eradicate downtime and maintain application h
 >
 > The [reference implementations](mission-critical-overview.md#illustrative-examples) are part of an open source project available on GitHub. The code assets illustrate considerations and recommendations for acheiving optimal CI/CD pipelines for a mission-critical application.
 
+## Demo: End-to-end resiliency testing with Azure Load Test and Azure Chaos Studio
```

Comment:
Can we change the text ("End-to-end resiliency testing with Azure Load Test and Azure Chaos Studio") to read: "Continuous validation and testing with Azure Load Test and Azure Chaos Studio"